light_prop/utils.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import numpy as np
+from PIL import Image
+
+
+def image_to_amplitude(image_path):
+    array = ndarray_to_image(image_path)
+    return normalize_array(array, 1)
+
+
+def image_to_phase(image_path):
+    array = ndarray_to_image(image_path)
+    return normalize_array(array, 2 * np.pi)
+
+
+def amplitude_to_image(array: np.array, file_path="output", file_extension="bmp"):
+    array = normalize_array(array, 255)
+    ndarray_to_image(array, file_path, file_extension)
+
+
+def phase_to_image(array: np.array, file_path="output", file_extension="bmp"):
+    array = array % 2 * np.pi
+    array = normalize_array(array, 255)
+    ndarray_to_image(array, file_path, file_extension)
+
+
+def ndarray_to_image(array: np.array, file_path="output", file_extension="bmp"):
+    path = f"{file_path}.{file_extension}"
+    logging.info("Reading array from ")
+    im = Image.fromarray(array)
+
+    if im.mode != 'RGB':
```

Comment:
I think that we do not need RGB at all right? We are saving one channel only right? So I'd rather have default 16bit png, BMP can only have 8bit. 
I know that reading BMP binary is easier though since its not compressed.

Reply:
I will look into this